README.pl.md

```diff
@@ -0,0 +1,147 @@
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_06.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_06.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_07.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_07.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_08.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_08.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_09.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_09.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png)
+[<img src="fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png" width=405>](fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png)
+[<img src="fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png" width=405>](fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png)
+
+## Opis
+
+NewPipe nie używa żadnych bibliotek Google ani API YouTube. Strony są jedynie analizowane aby zdobyć wymagane informacje, dzięki czemu ta aplikacja może zostać zainstalowana na urządzeniach bez Usług Google Play. Konto YouTube nie jest wymagane aby używać NewPipe, które jest wolnym oprogramowaniem copyleft.
+
+### Funkcje
+
+* Wyszukiwanie filmów
+* Brak wymogu konta Google/YouTube
+* Wyświetlanie generalnych informacji o filmach
+* Oglądanie filmów na YouTube
+* Słuchanie filmów na YouTube
+* Tryb okienkowy (Pływające okno)
+* Wybieranie odtwarzacza przez który możemy oglądać film
+* Pobieranie filmów
+* Pobieranie audio
+* Otwieranie filmów w Kodi
+* Pokazanie następnych/podobnych filmów
+* Wyszukiwanie w YouTube w konkretnym języku
+* Oglądanie/Blokowanie materiałów zablokowanych wiekowo.
+* Wyświetlanie generalnych informacji o kanałach
+* Wyszukiwanie kanałów
+* Oglądanie filmów ze strony kanału
+* Wsparcie dla Orbot/Tor (Jeszcze nie bezpośrednio)
+* Wsparcie dla 1080p/2K/4K
+* Historia oglądania
+* Subskrybowanie kanałów
+* Historia wyszukiwania
+* Wyszukiwanie/Oglądanie playlist
+* Oglądanie playlist w kolejce
+* Dodawnaie filmów do kolejki
+* Lokalne playlisty
+* Napisy
+* Wsparcie dla livestream'ów
+* Wyświetlanie komentarzy
+
+### Wspierane usługi
+
+NewPipe wspiera wiele usług. Nasza [dokumentacja](https://teamnewpipe.github.io/documentation/) dostarcza więcej informacji o tym jak nowe usługi mogą zostać dodane do aplikacji i extractora. Proszę skontaktuj się z nami jeśli masz zamiar dodać jakąś nową usługę. Obecnie wspierane usługi to:
+
+* YouTube
+* SoundCloud \[beta\]
+* media.ccc.de \[beta\]
+* Instancje PeerTube \[beta\]
+* Bandcamp \[beta\]
+
+<!-- Hidden span to keep old links compatible. -->
+<span id="updates"></span>
+

```

Comment:
That's not necessary here; there aren't any old links to the Polish README because it's not even merged in yet. :)
```suggestion
```